src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -263,4 +267,14 @@ private void validateFilingDate(LocalDate filingDate, Errors errors, String logg
             ApiLogger.infoContext(loggingContext, errorMessage);
         }
     }
+
+    private void validateHasSoldLand(Boolean hasSoldLand, Errors errors, String loggingContext) {
+        if (Boolean.TRUE.equals(hasSoldLand)) {
```

Comment:
After discussion with @lduranteau realised that this needs to also check a null value isn't supplied (was originally written thinking this logic had to cope with Update/Remove submissions too, but actually not the case).

Reply:
does the 'has_sold_land' appear in all journeys? i though it was missing in some, in which case we would still need to handle a null if the user is not presented with the question/screen.

Reply:
Doesn't appear on the Update/Remove journeys but then this validation method isn't called on those journeys so we need the flag I think.

Reply:
Feature flag now used.